engine/table/src/main/java/io/deephaven/engine/util/AbstractScriptSession.java

```diff
@@ -325,6 +325,10 @@ private ScriptSessionQueryScope(ScriptSession scriptSession) {
         public void putObjectFields(Object object) {
             throw new UnsupportedOperationException();
         }
+
+        public ScriptSession scriptSession() {
+            return this.scriptSession;
```

Comment:
No need for `this.` in this case.
```suggestion
            return scriptSession;
```